mlrun/runtimes/nuclio/application/application.py

```diff
@@ -480,20 +481,33 @@ def create_api_gateway(
 
     def invoke(
         self,
-        path: str,
+        path: str = "",
         body: typing.Optional[typing.Union[str, bytes, dict]] = None,
         method: str = None,
         headers: dict = None,
         dashboard: str = "",
         force_external_address: bool = False,
         auth_info: schemas.AuthInfo = None,
         mock: bool = None,
+        credentials: tuple[str, str] = None,
         **http_client_kwargs,
     ):
         self._sync_api_gateway()
+
+        if not credentials and auth_info:
+            credentials = (auth_info.username, auth_info.password)
+
         # If the API Gateway is not ready or not set, try to invoke the function directly (without the API Gateway)
         if not self.status.api_gateway:
-            super().invoke(
+            # create a requests auth object if credentials are provided
+            auth = (
```

Comment:
this blocks providing nuclio auth info in `http_client_kwargs` (for access key for example). can do something like:
```
auth = http_client_kwargs.pop("auth") or ...
```